### PR TITLE
migrate/push: allow skipping the 'latest' tag

### PR DESCRIPTION
**Pull request by noamcattan**

### CODE REVIEW

File: atlasaction/action.go [hunk 1]
```diff
@@ -289,9 +289,16 @@ func MigratePush(ctx context.Context, client *atlasexec.Client, act Action) erro
...
+	switch latest := act.GetInput("latest"); latest {
+	case "false":
+	case "true":
+		// Push the "latest" tag.
+		_, err = client.MigratePush(ctx, params)
+		if err != nil {
+			return fmt.Errorf("failed to push directory: %v", err)
+		}
+	default:
+		return fmt.Errorf(`invalid value for input "latest": only "true" or "false" are allowed, got %q`, latest)
```
Comment (a8m) [inline atlasaction/action.go:301]:
```suggestion
	switch latest, err := strconv.ParseBool(act.GetInput("latest")); {
	case err != nil:
			return fmt.Errorf(`invalid value for input "latest": only "true" or "false" are allowed, got %q`, latest)
	case latest:
		// Push the "latest" tag.
		_, err = client.MigratePush(ctx, params)
		if err != nil {
			return fmt.Errorf("failed to push directory: %v", err)
		}
```

**Review by a8m — COMMENTED** [review]
Can you update the doc with a small example? https://github.com/ariga/atlas-action?tab=readme-ov-file#arigaatlas-actionmigratepush

File: atlasaction/action.go (hunk 1 above)
Comment (noamcattan) [reply atlasaction/action.go:301]: I prefer it to be just "true" or "false".

**Comment by noamcattan** [comment]
> Can you update the doc with a small example? https://github.com/ariga/atlas-action?tab=readme-ov-file#arigaatlas-actionmigratepush

I updated the readme

File: atlasaction/action.go (hunk 1 above)
Comment (a8m) [reply atlasaction/action.go:301]: kk

**Review by a8m — APPROVED** [review]

**Pull request merged**